build fail on macos 12

### Description

try to use colima, but build fail

```text
version:1
:debug:main Starting logging for lima @2.0.1_0
:debug:sysinfo macOS 12.6 (darwin/21.6.0) arch i386
:debug:sysinfo MacPorts 2.11.6
:debug:sysinfo Xcode none, CLT 14.2.0.0.1.1668646533
:debug:sysinfo SDK 12
:debug:sysinfo MACOSX_DEPLOYMENT_TARGET: 12.0
:debug:main Attempting ln -sf /opt/local/var/macports/build/lima-d90d8b7b/work /opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/sysutils/lima/work
:debug:main dropping privileges: euid changed to 502, egid changed to 501.
:debug:main Executing org.macports.main (lima)
:debug:main Privilege de-escalation not attempted as not running as root.
:debug:archivefetch archivefetch phase started at Thu Nov 27 13:22:12 CST 2025
:msg:archivefetch --->  Fetching archive for lima
:debug:archivefetch Executing org.macports.archivefetch (lima)
:debug:archivefetch euid/egid changed to: 0/0
:debug:archivefetch chowned /opt/local/var/macports/incoming to macports
:debug:archivefetch euid/egid changed to: 502/501
:info:archivefetch --->  lima-2.0.1_0.darwin_21.x86_64.tbz2 doesn't seem to exist in /opt/local/var/macports/incoming/verified
:msg:archivefetch --->  Attempting to fetch lima-2.0.1_0.darwin_21.x86_64.tbz2 from https://mirror.sjtu.edu.cn/macports/packages/lima
:debug:archivefetch Fetching archive failed: The requested URL returned error: 403
:msg:archivefetch --->  Attempting to fetch lima-2.0.1_0.darwin_21.x86_64.tbz2 from https://pek.cn.packages.macports.org/macports/packages/lima
:debug:archivefetch Fetching archive failed: The requested URL returned error: 404
:msg:archivefetch --->  Attempting to fetch lima-2.0.1_0.darwin_21.x86_64.tbz2 from https://kmq.jp.packages.macports.org/lima
:debug:archivefetch Fetching archive failed: The requested URL returned error: 404
:debug:archivefetch Privilege de-escalation not attempted as not running as root.
:debug:fetch fetch phase started at Thu Nov 27 13:22:14 CST 2025
:notice:fetch --->  Fetching distfiles for lima
:debug:fetch Executing org.macports.fetch (lima)
:debug:fetch Privilege de-escalation not attempted as not running as root.
:debug:checksum checksum phase started at Thu Nov 27 13:22:14 CST 2025
:notice:checksum --->  Verifying checksums for lima
:debug:checksum Executing org.macports.checksum (lima)
:info:checksum --->  Checksumming lima-2.0.1.tar.gz
:debug:checksum Calculated (rmd160) is b406106ac51bb035f56795249cc21746a3ae9c48
:debug:checksum Correct (rmd160) checksum for lima-2.0.1.tar.gz
:debug:checksum Calculated (sha256) is a3efa067676ca80e780671eade074a5ff8ea080b04563f3cfd07cfc9ca4cbf76
:debug:checksum Correct (sha256) checksum for lima-2.0.1.tar.gz
:debug:checksum Calculated (size) is 7726111
:debug:checksum Correct (size) checksum for lima-2.0.1.tar.gz
:debug:checksum Privilege de-escalation not attempted as not running as root.
:debug:extract extract phase started at Thu Nov 27 13:22:14 CST 2025
:notice:extract --->  Extracting lima
:debug:extract Executing org.macports.extract (lima)
:info:extract --->  Extracting lima-2.0.1.tar.gz
:debug:extract setting option extract.args to '/opt/local/var/macports/distfiles/go/lima-2.0.1.tar.gz'
:debug:extract Environment: 
:debug:extract CC_PRINT_OPTIONS='YES'
:debug:extract CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/lima-d90d8b7b/work/.CC_PRINT_OPTIONS'
:debug:extract CPATH='/opt/local/include'
:debug:extract DEVELOPER_DIR='/Library/Developer/CommandLineTools'
:debug:extract LIBRARY_PATH='/opt/local/lib'
:debug:extract MACOSX_DEPLOYMENT_TARGET='12.0'
:debug:extract MACPORTS_LEGACY_SUPPORT_DISABLED='1'
:debug:extract SDKROOT='/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk'
:debug:extract SOURCE_DATE_EPOCH='1762882628'
:info:extract Executing:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/go/lima-2.0.1.tar.gz' | /usr/bin/tar -xf - 
:debug:extract system:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/go/lima-2.0.1.tar.gz' | /usr/bin/tar -xf - 
:debug:extract euid/egid changed to: 0/0
:debug:extract chowned /opt/local/var/macports/build/lima-d90d8b7b/work to macports
:debug:extract euid/egid changed to: 502/501
:debug:extract Executing proc-post-org.macports.extract-extract-0
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc/usr
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc/usr/bin
:debug:extract compiler_wrapper: Creating /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc/usr/bin/clang
:debug:extract compiler_wrapper:  -> Comp Flags: Will embed ' -Os' in cc wrapper script
:debug:extract compiler_wrapper:  -> SDK: Will embed '-isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk' in cc wrapper script
:debug:extract compiler_wrapper:  -> Will embed -pipe in cc wrapper script
:debug:extract compiler_wrapper:  -> Will embed legacysupport flags in cc wrapper script
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objc
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objc/usr
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objc/usr/bin
:debug:extract compiler_wrapper: Creating /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objc/usr/bin/clang
:debug:extract compiler_wrapper:  -> Comp Flags: Will embed ' -Os' in objc wrapper script
:debug:extract compiler_wrapper:  -> SDK: Will embed '-isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk' in objc wrapper script
:debug:extract compiler_wrapper:  -> Will embed -pipe in objc wrapper script
:debug:extract compiler_wrapper:  -> Will embed legacysupport flags in objc wrapper script
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cxx
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cxx/usr
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cxx/usr/bin
:debug:extract compiler_wrapper: Creating /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cxx/usr/bin/clang++
:debug:extract compiler_wrapper:  -> Comp Flags: Will embed ' -Os -stdlib=libc++' in cxx wrapper script
:debug:extract compiler_wrapper:  -> SDK: Will embed '-isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk' in cxx wrapper script
:debug:extract compiler_wrapper:  -> Will embed -pipe in cxx wrapper script
:debug:extract compiler_wrapper:  -> Will embed legacysupport flags in cxx wrapper script
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objcxx
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objcxx/usr
:info:extract xinstall: mkdir /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objcxx/usr/bin
:debug:extract compiler_wrapper: Creating /opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objcxx/usr/bin/clang++
:debug:extract compiler_wrapper:  -> Comp Flags: Will embed ' -Os -stdlib=libc++' in objcxx wrapper script
:debug:extract compiler_wrapper:  -> SDK: Will embed '-isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk' in objcxx wrapper script
:debug:extract compiler_wrapper:  -> Will embed -pipe in objcxx wrapper script
:debug:extract compiler_wrapper:  -> Will embed legacysupport flags in objcxx wrapper script
:debug:extract Executing proc-post-org.macports.extract-extract-1
:debug:extract Executing proc-post-org.macports.extract-extract-2
:debug:extract Privilege de-escalation not attempted as not running as root.
:debug:patch patch phase started at Thu Nov 27 13:22:14 CST 2025
:debug:patch Executing org.macports.patch (lima)
:notice:patch --->  Applying patches to lima
:info:patch --->  Applying patch-Makefile.diff
:debug:patch Environment: 
:debug:patch CC_PRINT_OPTIONS='YES'
:debug:patch CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/lima-d90d8b7b/work/.CC_PRINT_OPTIONS'
:debug:patch CPATH='/opt/local/include'
:debug:patch DEVELOPER_DIR='/Library/Developer/CommandLineTools'
:debug:patch LIBRARY_PATH='/opt/local/lib'
:debug:patch MACOSX_DEPLOYMENT_TARGET='12.0'
:debug:patch SDKROOT='/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk'
:debug:patch SOURCE_DATE_EPOCH='1762882628'
:info:patch Executing:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && /usr/bin/patch -t -N -p0 < '/opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/sysutils/lima/files/patch-Makefile.diff'
:debug:patch system:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && /usr/bin/patch -t -N -p0 < '/opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/sysutils/lima/files/patch-Makefile.diff'
:info:patch patching file Makefile
:info:patch --->  Applying patch-usrlocalgo.diff
:debug:patch Environment: 
:debug:patch CC_PRINT_OPTIONS='YES'
:debug:patch CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/lima-d90d8b7b/work/.CC_PRINT_OPTIONS'
:debug:patch CPATH='/opt/local/include'
:debug:patch DEVELOPER_DIR='/Library/Developer/CommandLineTools'
:debug:patch LIBRARY_PATH='/opt/local/lib'
:debug:patch MACOSX_DEPLOYMENT_TARGET='12.0'
:debug:patch SDKROOT='/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk'
:debug:patch SOURCE_DATE_EPOCH='1762882628'
:info:patch Executing:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && /usr/bin/patch -t -N -p0 < '/opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/sysutils/lima/files/patch-usrlocalgo.diff'
:debug:patch system:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && /usr/bin/patch -t -N -p0 < '/opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/sysutils/lima/files/patch-usrlocalgo.diff'
:info:patch patching file pkg/usrlocal/usrlocal.go
:debug:patch Executing proc-post-org.macports.patch-patch-0
:info:patch --->  Patching Makefile: s|@@VERSION@@|2.0.1|g
:debug:patch Executing reinplace: /usr/bin/sed s|@@VERSION@@|2.0.1|g </opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima/Makefile >@file15
:debug:patch euid/egid changed to: 0/0
:debug:patch chowned /opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima/Makefile to macports
:debug:patch euid/egid changed to: 502/501
:debug:patch euid/egid changed to: 0/0
:debug:patch setting attributes on /opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima/Makefile
:debug:patch euid/egid changed to: 502/501
:debug:patch Privilege de-escalation not attempted as not running as root.
:debug:configure configure phase started at Thu Nov 27 13:22:14 CST 2025
:notice:configure --->  Configuring lima
:debug:configure Preferred compilers: clang macports-clang-17 macports-clang-16 macports-clang-15 macports-clang-14 macports-clang-13 macports-clang-12 macports-clang-11 macports-clang-10 macports-clang-9.0
:debug:configure Using compiler 'Xcode Clang'
:debug:configure Executing org.macports.configure (lima)
:debug:configure Executing portconfigure::configure_finish
:debug:configure Privilege de-escalation not attempted as not running as root.
:debug:build build phase started at Thu Nov 27 13:22:14 CST 2025
:notice:build --->  Building lima
:debug:build Executing org.macports.build (lima)
:debug:build Environment: 
:debug:build CC='/opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc/usr/bin/clang'
:debug:build CC_PRINT_OPTIONS='YES'
:debug:build CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/lima-d90d8b7b/work/.CC_PRINT_OPTIONS'
:debug:build CPATH='/opt/local/include'
:debug:build CXX='/opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cxx/usr/bin/clang++'
:debug:build DEVELOPER_DIR='/Library/Developer/CommandLineTools'
:debug:build F77=''
:debug:build F90=''
:debug:build FC=''
:debug:build GOARCH='amd64'
:debug:build GOOS='darwin'
:debug:build GOPATH='/opt/local/var/macports/build/lima-d90d8b7b/work/gopath'
:debug:build LIBRARY_PATH='/opt/local/lib'
:debug:build MACOSX_DEPLOYMENT_TARGET='12.0'
:debug:build MACPORTS_LEGACY_SUPPORT_DISABLED='1'
:debug:build OBJC='/opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objc/usr/bin/clang'
:debug:build OBJCXX='/opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/objcxx/usr/bin/clang++'
:debug:build SDKROOT='/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk'
:debug:build SOURCE_DATE_EPOCH='1762882628'
:info:build Executing:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && make -j16 -w native 
:debug:build system:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && make -j16 -w native 
:info:build Guestagents are unzipped each time to check the build configuration; they may be gunzipped afterward.
:info:build make: Entering directory `/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima'
:info:build rm -rf _output vendor
:info:build CGO_ENABLED=1 GOOS="darwin" GOARCH="amd64" CC="/opt/local/var/macports/build/lima-d90d8b7b/work/compwrap/cc/usr/bin/clang" go build -ldflags="-s=true -w=true -X github.com/lima-vm/lima/v2/pkg/version.Version=2.0.1" -tags=,no_vz -tags 'no_vz' -o _output/bin/limactl ./cmd/limactl
:info:build go: downloading al.essio.dev/pkg/shellescape v1.6.0
...
:info:build pkg/driver/vz/register.go:11:20: undefined: New
:info:build make: *** [_output/bin/limactl] Error 1
:info:build make: Leaving directory `/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima'
:info:build Command failed:  cd "/opt/local/var/macports/build/lima-d90d8b7b/work/gopath/src/github.com/lima-vm/lima" && make -j16 -w native 
:info:build Exit code: 2
:error:build Failed to build lima: command execution failed
:debug:build Error code: CHILDSTATUS 1748 2
:debug:build Backtrace: command execution failed
:debug:build     while executing
:debug:build "system {*}$notty {*}$callback {*}$nice $fullcmdstring"
:debug:build     invoked from within
:debug:build "command_exec -callback portprogress::target_progress_callback build"
:debug:build     (procedure "portbuild::build_main" line 10)
:debug:build     invoked from within
:debug:build "$procedure $targetname"
:error:build See /opt/local/var/macports/logs/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_sysutils_lima/lima/main.log for details.

:debug:build "system {*}$notty {*}$callback {*}$nice $fullcmdstring"
:debug:build     invoked from within
:debug:build "command_exec -callback portprogress::target_progress_callback build"
:debug:build     (procedure "portbuild::build_main" line 10)
:debug:build     invoked from within
:debug:build "$procedure $targetname"
:error:build See /opt/local/var/macports/logs/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_sysutils_lima/lima/main.log for details.

```

--- a/pkg/driver/vz/register.go
+++ b/pkg/driver/vz/register.go
@@ -1,4 +1,4 @@
-//go:build darwin && !external_vz
+//go:build darwin && !no_vz && !external_vz
 
 // SPDX-FileCopyrightText: Copyright The Lima Authors
 // SPDX-License-Identifier: Apache-2.0

